magicctapipe/io/io.py

```diff
@@ -217,6 +217,23 @@ def recursive_solution(current_tel, current_comb):
 
     for key in range(len(keys)):
         recursive_solution(key, ["", []])
+    values = list(TEL_NAMES.values())  # TODO: remove in next PR
```

Comment:
ok, you really resuffled here :-). I've meant something easier, similar to what you were doing in the previous PR:
if set(values) = ....
  TEL_COMBINATIONS = {
    "M1_M2": [2, 3],  # combo_type = 0
    "LST1_M1": [1, 2],  # combo_type = 1
    "LST1_M2": [1, 3],  # combo_type = 2
    "LST1_M1_M2": [1, 2, 3],  # combo_type = 3
} 
the current code, even if just temporary it is still pretty ugly to read. It must work however, because you have a test on LST1+M1+M2 combination. Either way, no matter if you change it or not, let's merge this PR